New gs english (#2497)
tarantool/doc: doc/getting_started/writing_cluster_code.rst

It is also included in the tutorial default application.

However, in Tarantool, you can implement absolutely any business logic on top of a cluster.

> The :ref:`Cartridge developer getting started <getting_started_cartridge>` section is covers cluster roles mechanism and writing a cluster application from scratch.

Reviewer: ```suggestion
This :doc:`Cartridge tutorial section </getting_started/getting_started_cartridge>` covers the cluster roles mechanism and writing a cluster application from scratch.
```
